deployment/prerequisites.ts

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+import "source-map-support/register";
+import {
+  aws_apigateway as apigateway,
+  aws_iam as iam,
+} from "aws-cdk-lib";
```

Comment:
```suggestion
App,
   DefaultStackSynthesizer,
   Stack,
   StackProps,
   Tags,
} from "aws-cdk-lib";
```

Reply:
I was trying to suggest a change, but then I messed up the formatting.  I ended up making a commit.